daemon/api_snaps.go

```diff
@@ -765,3 +802,73 @@ func shouldSearchStore(r *http.Request) bool {
 
 	return false
 }
+
+func snapHoldMany(inst *snapInstruction, st *state.State) (res *snapInstructionResult, err error) {
+	if err := validateHoldTime(inst.Time); err != nil {
+		return nil, err
+	}
+
+	patchValues := map[string]interface{}{"refresh.hold": inst.Time}
+	var msg string
+	var ts *state.TaskSet
+
+	if len(inst.Snaps) == 0 {
+		ts, err = configstate.ConfigureInstalled(st, "core", patchValues, 0)
+		if err != nil {
+			return nil, err
+		}
+
+		msg = i18n.G("Held refreshes for all snaps.")
+	} else {
+		ts, err = snapstate.CreateGateRefreshesTask(st, inst.Time, inst.Snaps)
+		if err != nil {
+			return nil, err
+		}
+
+		msg = fmt.Sprintf(i18n.G("Held refreshes for %s."), strutil.Quoted(inst.Snaps))
```

Comment:
s/Held/Hold/ ? the summary are usually in the imperative. Same for the other summaries